### DISCUSSION TITLE
Output location on windows not worked

### DISCUSSION THREAD

**Discussion by viniciusaffonso-hotmart** [prisma/prisma#21165] [Q&A]
### Bug description

When I try to use the prisma generator with a custom location on Windows, some file has a relative path and the error below occurs

<img width="584" alt="image" src="https://github.com/prisma/prisma/assets/89027484/8aa9a93a-e5c7-4283-9b1e-0aeab9858b19">

### How to reproduce

npx prisma generate
then 
yarn build

### Expected behavior

Run custom output with success

### Prisma information
```prisma
generator client {
  provider      = "prisma-client-js"
  output        = "prisma-client"
  binaryTargets = ["native"]
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model User {
  id       Int        @id @default(autoincrement())
  email    String     @unique
  name     String
  Password Password[]
}



model Password {
  id      Int       @id @default(autoincrement())
  email   String    @unique
  userId  Int
  created DateTime  @default(now())
  updated DateTime?
  user    User      @relation(fields: [userId], references: [id])
}





```

### Environment & setup

- OS: Windows 11
- Database: PostgreSQL
- Node.js version: v16.17.0

### Prisma Version

```
5.3.1
```

**Comment by ludralph** [comment]
Hi @viniciusaffonso-hotmart 👋 

Thank you for raising this question.

Do you have the necessary permissions to write to the directory where you're trying to generate the Prisma client? You might need to run your command prompt or terminal as an administrator. Another thing you can do is to stop the Nextj.s server and try this command again.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by viniciusaffonso-hotmart** [reply]
The output file is generated. The problem is someone file use relative path. 
Windows can't resolve this relative path.
The issue occur with yarn build command on nextjs

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
